packages/allow-scripts/src/config.js

```diff
@@ -0,0 +1,358 @@
+const { existsSync, promises: fs } = require('node:fs')
+const path = require('node:path')
+const { FEATURE } = require('./toggles')
+const normalizeBin = require('npm-normalize-package-bin')
+const { loadCanonicalNameMap } = require('@lavamoat/aa')
+
+const bannedBins = new Set(['corepack', 'node', 'npm', 'pnpm', 'yarn'])
+
+/**
+ * @typedef PkgLavamoatConfig
+ * @property {Record<string, any>} [allowScripts]
+ * @property {Record<string, any>} [allowBins]
+ * @property {Record<string, any>} [allowConfig]
+ * @property {Record<string, any>} [allowedPatterns]
+ * @property {Record<string, any>} [disallowedPatterns]
+ * @property {Record<string, any>} [missingPolicies]
+ * @property {Record<string, any>} [excessPolicies]
+ */
+/**
+ * @typedef {import('type-fest').PackageJson & { lavamoat: PkgLavamoatConfig }} LavamoatPackageJson
```

Comment:
please use the new [@import tag](https://devblogs.microsoft.com/typescript/announcing-typescript-5-5/#the-jsdoc-@import-tag)--it's way nicer

Reply:
I'm just copying over your code from f10ac687 > 82f9d3 - can we not address that in a separate PR?